front/src/applications/osrd/components/DriverTrainSchedule/DriverTrainScheduleModal.js

```diff
@@ -41,7 +46,7 @@ export default function DriverTrainScheduleModal(props) {
           <div className="row no-gutters">
             <div className="col-4 col-xl-5">{t('drivertrainschedule:mass')}</div>
             <div className="font-weight-bold text-primary col-8 col-xl-7">
-              {rollingStockSelected.mass}T
+              {Math.round(rollingStockSelected.mass / 100) / 10}T
```

Comment:
What does that do ?
make a function that abstracts the behavior with an explicit name that says what it does.

Reply:
It represents mass with one decimal. Function created, done  ;)